src/modules/FunModule.cpp

```diff
@@ -62,4 +72,712 @@ FunModule::FunModule() : Module("funutil", true), m_memeChannel(0)
 
 		});
 
+	RegisterCommand(Commands::FUN_POLL, "poll", [this](Discord::Client& client, const Discord::Message& message, const Discord::Channel& channel) 
+	{
+
+		QStringList lines = message.content().split('\n');
+
+		for (auto& line : lines) 
+		{
+			line = line.trimmed();
+		}
+
+		QStringList args = lines.at(0).simplified().split(' ');
+		GuildSetting* setting = &GuildSettings::GetGuildSetting(channel.guildId());
+		QString prefix = setting->prefix;
+
+		
+
+		if (args.first() != prefix + "poll")
+			return;
```

Comment:
Doesn't the command handler handle this for us?

Reply:
Ah well I'm pretty sure `LevelModule.cpp` did that for some reason, so I thought it would better to conform to that...
(All other commands in `CurrencyModule.cpp` do that as well)